Storage/SQL/SQLiteHistory.swift

```diff
@@ -283,6 +283,10 @@ extension SQLiteHistory: BrowserHistory {
             site.latestVisit = Visit(date: latest, type: VisitType.Unknown)
         }
 
+        if let bookmarked = row["bookmarked"] as? Int {
```

Comment:
Perhaps add an optional variant to `extension SDRow.getBoolean` elsewhere in this file?